docs/Release_Notes.gfs.v16.3.14.md

```diff
@@ -60,17 +62,25 @@ SORC CHANGES
 JOBS CHANGES
 ------------
 
-* No changes from GFS v16.3.13
+* Added a new `j-job` `JGDAS_ATMOS_ANALYSIS_WDQMS`
 
 PARM/CONFIG CHANGES
 -------------------
 
-* No changes from GFS v16.3.13
+* Added a `config.wdqms` file for the new j-job added.
+* Added new configuration parameter `DO_WDQMS` and set it to `YES` in the following configuration files
+  - `config.base.emc.dyn`
```

Comment:
```suggestion
```
I think it is a `NO` in `config.base.emc.dyn`.
It should be noted that `config.base.emc.dyn` was also updated.

Reply:
True!!  I will modify it.
Thanks!